Competitive_Programming/C++/Bit Manipulation/bit_manipulation.md

```diff
@@ -0,0 +1,105 @@
+# Bit Manipulation
+
+## Use of Bit Operators
+
+Bit operators perform operations much faster than arthmetic or logical operators
+
+## Types of Bit Operator
+
+There are mainly 6 types of bit operators
+
+1. `AND`
+2. `OR`
+3. `XOR`
+4. `Left shift operator`
+5. `Right shift operator`
+6. `Bitwise NOT`
+
+   <br/><br>
+
+   ![Symbols representing Bitwise operator](https://miro.medium.com/max/540/1*lwKj-TpvToBxuq98LXII1A.png)
+
+<br/><br>
+
+## **_Bitwise AND_**
+
+The bitwise AND operator takes two arguments and performs AND operation on the operands bit by bit.
+The operation will only be 1 when both operands are 1 otherwise it remains 0.
+
+![Table](https://i1.wp.com/techvidvan.com/tutorials/wp-content/uploads/sites/2/2019/12/implementation-of-and-operation-on-binary-digits.jpg?resize=341%2C272&ssl=1)
+
+<br />
+
+## **_Bitwise OR_**
+
+The bitwise OR operator takes two arguments and performs OR operation on each bit of the value of the operand. In the OR operation in binary, the resultant is 0 when both operands are 0 otherwise it is 1.
```

Comment:
```suggestion
The bitwise OR operator takes two arguments and performs OR operations on each bit of the operands. In the OR operation, the resultant is 0 when i-th bit of both the operands is 0 otherwise the result is 1.
```